cmd/otelcol/config/collector/splunk_config_non_linux.yaml

```
@@ -77,7 +74,7 @@ service:
     traces:
       receivers: [jaeger, opencensus, otlp, sapm, zipkin]
       processors: [memory_limiter, batch]
-      exporters: [sapm, signalfx_correlation]
+      exporters: [sapm]
```

signalfx exporter can go in traces pipeline now for correlation:

```suggestion
      exporters: [sapm, signalfx]
```